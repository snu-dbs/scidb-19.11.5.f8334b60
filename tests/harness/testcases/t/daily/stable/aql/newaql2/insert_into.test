--setup
create array A1 <a:int32> [x=0:2,3,0]
create array A2 <b:int32> [y=0:2,2,0]

--test
--aql insert into A1 '[(1)()()]'
scan(A1)  -- Inserted by scan_doctor
--aql insert into A2 '[()()][(3)]'
scan(A2)  -- Inserted by scan_doctor
--aql insert into A1 select * from A2
scan(A1)  -- Inserted by scan_doctor

--cleanup
remove(A1)
remove(A2)
