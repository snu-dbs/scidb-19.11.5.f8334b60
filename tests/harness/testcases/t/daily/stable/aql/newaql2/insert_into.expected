Query was executed successfully

Query was executed successfully

Query was executed successfully

{x} a
{0} 1

Query was executed successfully

{y} b
{2} 3

Query was executed successfully

{x} a
{0} 1
{2} 3

Query was executed successfully

Query was executed successfully

